hazelcast/src/test/java/com/hazelcast/internal/util/phonehome/PhoneHomeCPSubsystemTest.java

```diff
@@ -107,36 +113,43 @@ public void testFencedLocksCount() {
         node.hazelcastInstance.getCPSubsystem().getLock("hazelcast@secondGroup").lock();
         node.hazelcastInstance.getCPSubsystem().getLock("hazelcast2@firstGroup").lock();
 
-        parameters = phoneHome.phoneHome(true);
-        assertThat(parameters.get(PhoneHomeMetrics.CP_FENCED_LOCKS_COUNT.getRequestParameterName())).isEqualTo("3");
+        assertTrueEventually(() -> {
+            Map<String, String> params = phoneHome.phoneHome(true);
+            assertThat(params.get(PhoneHomeMetrics.CP_FENCED_LOCKS_COUNT.getRequestParameterName())).isEqualTo("3");
+        });
     }
 
     @Test
     public void testCountdownLatchesCount() {
         Map<String, String> parameters;
         parameters = phoneHome.phoneHome(true);
-        assertThat(parameters.get(PhoneHomeMetrics.CP_COUNTDOWN_LATCHES_COUNT.getRequestParameterName())).isEqualTo("0");
+        assertThat(parameters.get(PhoneHomeMetrics.CP_COUNTDOWN_LATCHES_COUNT.getRequestParameterName()))
+                .isEqualTo("0");
 
         node.hazelcastInstance.getCPSubsystem().getCountDownLatch("hazelcast@firstGroup").trySetCount(1);
         node.hazelcastInstance.getCPSubsystem().getCountDownLatch("hazelcast@secondGroup").trySetCount(1);
         node.hazelcastInstance.getCPSubsystem().getCountDownLatch("hazelcast2@firstGroup").trySetCount(1);
 
-        parameters = phoneHome.phoneHome(true);
-        assertThat(parameters.get(PhoneHomeMetrics.CP_COUNTDOWN_LATCHES_COUNT.getRequestParameterName())).isEqualTo("3");
+        assertTrueEventually(() -> {
+            Map<String, String> params = phoneHome.phoneHome(true);
+            assertThat(params.get(PhoneHomeMetrics.CP_COUNTDOWN_LATCHES_COUNT.getRequestParameterName()))
+                    .isEqualTo("3");
+        });
     }
 
     @Test
     public void testAtomicLongsCount() {
-        Map<String, String> parameters;
-        parameters = phoneHome.phoneHome(true);
+        Map<String, String> parameters = phoneHome.phoneHome(true);
```

Comment:
Maybe it will be better to save the common pattern - separate `parameters` variable definition and separate initialisation, or make a single-line initialisation everywhere.